chart/epinio/templates/cert-secrets.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if not .Values.certManager.enabled }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-registry-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.registry.ca | quote }}
+  tls.key: {{ .values.certManager.registry.key | quote }}
+  tls.crt: {{ .values.certManager.registry.cert | quote }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.epinio.ca | quote }}
+  tls.key: {{ .values.certManager.epinio.key | quote }}
+  tls.crt: {{ .values.certManager.epinio.cert | quote }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-ca-root
```

Comment:
What is the purpose of this secret? Do we need it?

Reply:
It is, as far as I can tell, the root for the self-signed secrets (Issuer `selfsigned-issuer`).

Reply:
I believe this secret is not needed. The root CA is used only by CM for `epinio-ca` clusterIssuer when signing E domain certs.

Reply:
Ok. Will remove.

Reply:
Done. Commit c70af6671ffcfa3d79ada591394df28a1da18911.
Updated https://github.com/epinio/epinio/pull/2681 as well.